content/articles/premium-domains.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Premium Domains
+excerpt: In the new gTLDs domain space, a premium domain is a domain that has been considered of special value by a registrar.
+categories:
+- Domains
+---
+
+# Premium Domains
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+## What is a Premium Domain?
+
+A premium domain is a domain that has been considered of special value by a registrar. Premium domains generally are available in the new gTLDs domain space. They are usually more expensive that regular domains, prices going up to several thousand dollars for one domain in some cases.
+
+## What makes a Premium Domain valuable
+
+Premium domains appeal to specific businesses or groups of people providing them very with useful, specific domain names for their activities. Other times they are just popular keywords or a very well known combination of name and domain. Registrars also consider every two or three leter new gTLDs as premium, there is value on being very short.
```

Comment:
Would suggest: "Premium domains appeal to specific businesses or groups of people providing them with useful, specific domain names for their activities. Other times they are just popular keywords or a very well known combination of a name and domain. Some registries and registrars also consider two or three letter domains to be more valuable and therefore will be higher priced."